docs/src/Module.doc.js

```diff
@@ -159,4 +169,65 @@ function ModuleExample4() {
   />
 );
 
+card(
+  <Example
+    name="Multiple items with external control"
+    defaultCode={`
+function ModuleExample5() {
```

Comment:
can you copy paste & replace the example code (I just formated it the indentation)
```
function ModuleExample5() {
  const [extExpandedId, setExtExpandedId] = React.useState(null);
  return (
    <Box maxWidth={800} padding={2} column={12}>
      <Box display="flex" paddingY={2}>
        <Box paddingX={2}>
          <Button
            size="sm"
            text={extExpandedId === 'ModuleExample5-0' ? 'collapse 1' : 'expand 1'}
            onClick={() =>
              extExpandedId === 'ModuleExample5-0'
                ? setExtExpandedId('-1')
                : setExtExpandedId('ModuleExample5-0')
            }
          />
        </Box>
        <Box paddingX={2}>
          <Button
            size="sm"
            text={extExpandedId === 'ModuleExample5-1' ? 'collapse 2' : 'expand 2'}
            onClick={() =>
              extExpandedId === 'ModuleExample5-1'
                ? setExtExpandedId('-1')
                : setExtExpandedId('ModuleExample5-1')
            }
          />
        </Box>
        <Box paddingX={2}>
          <Button
            size="sm"
            text={extExpandedId === 'ModuleExample5-2' ? 'collapse 3' : 'expand 3'}
            onClick={() =>
              extExpandedId === 'ModuleExample5-2'
                ? setExtExpandedId('-1')
                : setExtExpandedId('ModuleExample5-2')
            }
          />
        </Box>
      </Box>
      <Module.Expandable
        id="ModuleExample5"
        accessibilityExpandLabel="Expand the module"
        accessibilityCollapseLabel="Collapse the module"
        extExpandedId={extExpandedId}
        setExtExpandedId={setExtExpandedId}
        items={[
          {
            title: 'Title1',
            summary: ['summary1'],
            children: <Text size="md">Children1</Text>,
          },
          {
            title: 'Title2',
            summary: ['summary2'],
            children: <Text size="md">Children2</Text>,
          },
          {
            title: 'Title3',
            summary: ['summary3'],
            children: <Text size="md">Children3</Text>,
          },
        ]}
      />
    </Box>
  );
}
```

Reply:
pasted

Reply:
Removed this example5 as requested.